Dockerfile

```diff
@@ -11,18 +11,44 @@
 
 FROM alpine:3.8
 
-RUN apk add --ino-cache git py3-pip npm py3-babel tini swig gcc alpine-sdk python3-dev
-
-RUN cd / && git clone -b master --depth=1 https://github.com/roramirez/qpanel.git && \
-    cd qpanel &&  pip3 install -r requirements.txt
-
-RUN cd /qpanel && npm install
-
-RUN cd /qpanel && pybabel compile -d qpanel/translations
-
-RUN cd /qpanel && cp samples/config.ini-dist config.ini
+RUN apk add --ino-cache \
+        git \
+        py3-pip \
+        npm \
+        py3-babel \
+        tini \
+        swig \
+        gcc \
+        alpine-sdk \
+        python3-dev && \
+    cd / && \
+    git clone \
+        -b master \
+        --depth=1 \
+        https://github.com/roramirez/qpanel.git && \
+    cd qpanel &&  \
+    pip3 install -r requirements.txt && \
+    cd /qpanel && \
+    npm install && \
+    cd /qpanel && \
+    pybabel compile -d qpanel/translations && \
+    cd /qpanel && \
+    cp samples/config.ini-dist config.ini && \
+    apk del --quiet \
+        git \
+        npm \
+        gcc \
+        alpine-sdk \
+        python-dev
 
 WORKDIR /qpanel
 
-CMD tini -- python3 app.py
+EXPOSE 5000
 
+CMD [ ! -z "${QPANEL_USER}"   ] && \
```

Comment:
These variables from where coming?. You could explain as comment in build.sh if  we'll need set the variables